### PR TITLE
not pretty yet, but basics are there

### PR DESCRIPTION
**Pull request by alnahra**

### CODE REVIEW

File: day_one.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+			<h2>About me</h2>
+			<p style='text-align:left;font-family:arial;'>I'm a student in St. Louis, Missouri.
+				I'm currently learning to code. I have various interested and stuff.</p>
+			<h2 style='text-align:left;font-family:arial;color:#00cc00;
+				border-width: 4px;border-bottom-style: dashed;border-bottom-color:#eee;'>My blog</h2>
+			<p style='text-align:left;font-family:arial;'>Here are some blog posts I've written</p>
+			<ul style='font-family:arial;'>
+					<li><a href='file:///Users/alianahra/Desktop/Fellowship/More%20info.html'>
+						More info about me</a></li>
+					<li><a href='file:///Users/alianahra/Desktop/Fellowship/weather.html'>
```
Comment (tilikang) [inline day_one.html]: I'm guessing you're not using this file anymore (feel free to delete it, git will always keep a backup so you can restore it if you need to). But if you are using it, these links won't work on your MAMP server

File: index.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<html>
+	<head>
+		<title>STL Municipality Research Project</title>
+		<link rel='stylesheet' href='style.css'/>
+		<link href="https://fonts.googleapis.com/css?family=Lalezar|Luckiest+Guy" rel="stylesheet">
+	</head>
+	<link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```
Comment (tilikang) [inline index.php]: This file has a `<head>` correctly, but not a `<body>`

File: practice/basic.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+		$sum = $num1 + $num2;
+		echo "Sum of the two numbers $num1 and $num2 is $sum";
+	}
+//call getSum function
+	getSum(15,35);
+//define array
+	$movie = array('title' => "Rear Window",
+					'director' => "Alfred Hitchcock",
+					'year' => "1954");
+	echo "the movie's title is $title"
```
Comment (tilikang) [inline practice/basic.php]: This line doesn't look like it would work.

File: calendar.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
+div#calendar{
```
[the remark this answers is not in the document]
Comment (alnahra) [reply calendar.css]: it's copied from elsewhere and I'm just messing around with it, but thank you! I commented in the url to the latest version for reference

**Pull request merged**